### DISCUSSION TITLE
Retrieve Helm chart versions from an OCI registry

### DISCUSSION THREAD

**Discussion by iverberk** [renovatebot/renovate#12507] [Request Help] [answered]
We are using Azure Container Registry as an OCI-based Helm chart repository. All our Helm charts are located here. I'd like to use a Regex Manager to scan that registry for newer versions of Helm charts. 

Is the Helm datasource capable of scanning an OCI-based registry for newer versions? Right now, I'm getting a 'Failed to look up dependency' error. Not sure which URL to configure, so that Renovate understands that it is dealing with an OCI registry vs regular one.

Thanks!

**Comment by secustor** [comment, marked as answer]
You have to use the `docker` datasource for OCI images.    
We do the same internally for the `helm3` manager. https://github.com/renovatebot/renovate/blob/main/lib/manager/helmv3/extract.ts#L81

**Reply by iverberk** [reply]
@secustor I will try that. Thanks for the response, it's appreciated!

**Reply by chinmaya-n** [reply]
Hi, that URL seems broken now and here is the new URL: https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/helmv3/extract.ts

Also, are there any example usages for this? I'm having trouble parsing using the regex customManager.

**Reply by secustor** [reply]
OCI helm chart references should work out of the box now. If that is not the case for you open a new discussion with a reproduction

**Reply by jankatins** [reply]
So the datasource should nowadays be "helm"?
